metadata-io/src/main/java/com/linkedin/metadata/search/elasticsearch/indexbuilder/SettingsBuilder.java

```diff
@@ -117,6 +119,7 @@ public class SettingsBuilder {
   public static final List<String> INDEX_TOKEN_FILTERS =  ImmutableList.of(
           ASCII_FOLDING,
           MULTIFILTER,
+          TRIM,
```

Comment:
Will this trigger a reindex?

Reply:
This was added, but also the stop words so 2 birds, one stone.